articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -8,11 +8,18 @@ You can also create a custom login for prompting the user for their username and
 
 ## Configure OpenID Connect Middleware
 
-The easiest way to enable authentication with Auth0 in your ASP.NET Core application is to use the OpenID Connect middleware. First, go to the `ConfigureServices` method of your `Startup` class and add the authentication services and enable cookie autentication by calling the `AddAuthentication` and `AddCookie` methods respectively. 
+To enable authentication in your ASP.NET Core application, use the OpenID Connect (OIDC) middleware.
+Go to the `ConfigureServices` method of your `Startup` class. To add the authentication services, call the `AddAuthentication` method. To enable cookie authentication, call the `AddCookie` method. 
 
-Next you can add a call to `AddOpenIdConnect` to configure the OpenID Connect authentication handler. Configure the authentication scheme by passing "Auth0" as the `authenticationScheme` parameter. This is important, as you will later on use this value to challenge the OIDC middleware.
+Next, configure the OIDC authentication handler. Add a call to `AddOpenIdConnect`. To configure the authentication scheme, pass "Auth0" as the `authenticationScheme` parameter. You will use this value later to test the OIDC middleware.
```

Comment:
> pass "Auth0" as the...

Should it not be 

> passing "Auth0" as the...

Reply:
> You will use this value later to test the...

No. **test** is not the correct word. It should be **challenge** as it originally was. You **challenge** authentication middleware, not **test** it.

So it should read:

> You will use this value later to challenge the...